src/main/resources/default/assets/wondergem/wondergem.css.pasta

```diff
@@ -2,8 +2,6 @@
  * A list of all licenses can be obtained via /licenses
  */
 
-<i:include name="/assets/wondergem/bootstrap/css/bootstrap.min.css"/>
-
```

Comment:
Is this change intended?

Reply:
Yes, the same file is included another time below in line 11 (previously line 13)